### ISSUE TITLE
Chelsea Ernhofer -- Broken Windows and Teen Birth Rates 

### ISSUE THREAD

**Issue by cernhofer** [uchicago-computation-workshop/2018_winter_conference#7]
🙌

**Comment by jfan3** [comment]
Do you link your multiple data sources down to the individual level?

Thanks!

**Comment by bensoltoff** [comment]
* Is this intended to be a causal explanation, or just a useful predictor?
* How useful is this predictor compared to alternative factors?
* Weird Canadian English
* Use a graph to present summary statistics - it's much more useful for a presentation format than trying to read a large table
* Any cross-validation used for the OLS model?
* Again, how much contribution does the neighbourhood order/disorder variable make in this model? (some of this is captured in the variance importance plot)

**Comment by rickecon** [comment]
I like your project. Good use of linear regression, decision trees, and random forests.

**Comment by jamesallenevans** [comment]
Physical environment sends direct message / ecological effect 
Linking data on demographic variables and perceived safety/disorder on teen pregnancy
At zip code level (why so high...why not Census block? (with ACS?)
Linear results presented
Why not over all urban areas in all cities...ACS linked with data on perceived safety
Super interesting / provocative association.

**Comment by bethbailey** [comment]
How do you control for the fact that built environments with a lot of disorder could have high correlations with other theorized causes for high teen birth rates?

**Comment by Alicechung** [comment]
I like the dataset you used, and applied deep learning and validate your result with different validation methods. It would be great if you can show the results with graphs :)

**Comment by w4rner** [comment]
Nice pres. Possible extension: how do rural physical environments compare to urban ones (both high & low disorder)?

**Comment by xiuyuanzhang** [comment]
This is really cool! Can you explain further how the social and ecological aspects are distinguished? It seems that the crime rate var belongs to the social aspect but included in your model?

**Comment by rodrigovaldes** [comment]
Beautiful presentation! I believe you need to convince the audience that you do not have a problem of omitted variables (everything correlated with your index of order).

**Comment by jmausolf** [comment]
Great, excited to see you using both regression and ML models. Thought with the regression relative to the outcome, if the outcomes are ordered survey response items, an ordered logistic regression or multinomial logit may be better depending on the dispersion of the DV.

**Comment by ruixue-li** [comment]
Have you tried doing some cross validation?

**Comment by cernhofer** [comment]
@jfan3 I WISH! Individual level data connected to geographical context is hard/impossible to come by. I actually spent most of my time last quarter trying to find individual level data that I could add to the model. Thanks for the comment!

**Comment by cernhofer** [comment]
@bensoltoff Thanks for the comment!
* Not causal- I originally planned to do pure prediction but realized that I'm interested in both predicting Birth rates as well as explaining the relationship between birth rates and built environment. So a little bit of both prediction and inference but I'm not claiming causality. 
* By alternative factors are you referring to social/individual level factors? Past literature has some results that indicate that ecological context is necessary
* Amazing Canadian English
* Yes, great comment- I completely overlooked using a graph for summary stats
* Yes cross-validation was used and the results were fairly consistent
* When doing the linear regression, including only neighbourhood disorder resulted in and R-squared of 0.11 so that gave me some indication that it has some importance but from the regression tree/random forest one can clearly see that it's not the most important variable in the model.

**Comment by cernhofer** [comment]
@jamesallenevans thanks for the comment!

Basically the answer to everything is data constraints- the perinatal (teen birth) data I have is at the zip code level and the StreetScore data is available only for New York City and Boston- it's my next step to expand to this second city.

**Comment by cernhofer** [comment]
@bethbailey- Basically just by including somer of those alternative approaches as controls in the model. I think that there's a large weakness in my analysis in that I have no way of looking at individual/family level variance which I think could have large influences. However, the high R-squared and significance of neighbourhood order in the linear model gives me some confidence, at least, that the work isn't pure garbage.

Thanks for the comment!

**Comment by cernhofer** [comment]
@Alicechung Great comment- thanks! You're totally right!

**Comment by cernhofer** [comment]
@lpwarner Thanks for the comment- that would be super interesting. Unfortunately, the specific way I'm operationalizing neighbourhood order/disorder (StreetScore) has only been applied to a few select urban areas in the States. If they ever make their algorithm public I would do that in an instant!

**Comment by cernhofer** [comment]
@xiuyuanzhang Thanks for the comment- I included crime mostly as a control for at least some form of social influence but you're correct in that it definitely creates confusion when I describe my model as purely ecological.

**Comment by cernhofer** [comment]
@rodrigovaldes Thanks!

**Comment by cernhofer** [comment]
@jmausolf I don't know if I understand your question. My response variable is a quantitative rate of teen birth. I'm not quite sure if there's an intuitive method whereby I could divide this variable into two groups in order to warrant a logistic regression.

**Comment by cernhofer** [comment]
@ruixue-li Yes! I didn't talk about it due to time constraints but I was able to do cross validation for the models and results were fairly consistent. Thanks for the question!

**Comment by RuxinChen** [comment]
While various neighborhood demographics (esp. disadvantage indicators) are controlled for, I am wondering if you have considered the effect of neighborhood gender and age compositions on teens birth rate. Presumably, a neighborhood consists of a higher proportion of teen males and younger teens might have a lower teen birth rate than one with more females and a more mature teen population. It would, therefore, be important to account for gender and age compositions at the neighborhood level.

**Comment by dpzhang** [comment]
What is your standardization methods so as to make each zip code area consistent?
